adage/templates/base.html

```diff
@@ -0,0 +1 @@
+index.html
```

Comment:
Mmm, great question - it should probably be a setting to make it more adaptable, like you say